src/rendererGLES/tr_image.c

```diff
@@ -793,8 +777,8 @@ static void Upload32(unsigned *data,
 				internalFormat = GL_RGB5;
 			}
 			else
-			{
-				internalFormat = GL_RGB;
+			{				
```

Comment:
For next commits, beware of trailing characters and tab/space settings. I took care of these for this time.